apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -228,10 +89,19 @@ storiesOf('Spinner converged', module)
     },
   )
   .addStory(
-    'Inverted with Huge',
+    'Inverted + size',
     () => (
       <InvertedWrapper>
-        <Spinner appearance="inverted" size="huge" />
+        <div className={useStyles().paused} style={{ display: 'flex', flexDirection: 'column', rowGap: '20px' }}>
+          <Spinner appearance="inverted" size="extra-tiny" label="Extra Tiny" />
+          <Spinner appearance="inverted" size="tiny" label="Tiny" />
+          <Spinner appearance="inverted" size="extra-small" label="Extra Small" />
+          <Spinner appearance="inverted" size="small" label="Small" />
+          <Spinner appearance="inverted" size="medium" label="Medium" />
+          <Spinner appearance="inverted" size="large" label="Large" />
+          <Spinner appearance="inverted" size="extra-large" label="Extra Large" />
+          <Spinner appearance="inverted" size="huge" label="Huge" />
+        </div>
       </InvertedWrapper>
     ),
```

Comment:
I mentioned this in a previous comment but I think it got lost. This `Inverted + size` story isn't necessary, since you've already tested the inverted appearance and size variants separately. Nothing in the inverted appearance changes based on size.